modules/face/samples/sampleKazemi.cpp

```diff
@@ -0,0 +1,124 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include "opencv2/face/face_alignment.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/imgcodecs.hpp"
+
+using namespace std;
+using namespace cv;
+
+int main(int argc,char** argv){
+    //Give the path to the directory containing all the files containing data
+    if(argc!=2){
+        String error_message = "No directory specified to give names of the files to be read\n Aborting....";
+        cerr<<error_message;
+        return 0;
+    }
+
+    String directory = argv[1];
+    //create a vector to store names of files in which annotations
+    // and image names are found
+    /*The format of the file containing annotations should be of following format
+        /data/abc
+        123.45,345.65
+        321.67,543.89
+
+        The above format is similar to HELEN dataset which is used for training model
+     */
+
+    vector<String> filenames;
+    //reading the files from the given directory
+    glob(directory,filenames);
+    //create a vector to store image names
+    vector<String> imagenames;
+    //create object to get landmarks
+    vector< vector<Point2f> > trainlandmarks;
+    //create a pointer to call the base class
+    Ptr<cv::face::FacemarkKazemi> facemark=cv::face::create();
```

Comment:
`cv::face::` can be reduced to `face::` but to make samples even more clear, you could just add `using namespace cv::face;` under the `using namespace cv;`